src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -112,6 +112,9 @@ failover_virtual_alias_address_tooltip: T('IP address to use for \
  administrative access to the system. The netmask <i>/32</i> is \
  required for this value and cannot be changed.'),
 
+failover_alias_set_error: T('Each alias must have an IP address for both\
+ controllers as well as the virtual interface.'),
```

Comment:
```suggestion
failover_alias_set_error: T('Each alias must have an IP address for each \
 controller and the virtual interface.'),
```

Reply:
I don't like the suggested verbiage change because the word `Each` is used twice in the same sentence which makes it redundant.

I'd rather use verbiage like `An IP address must be provided for each controller as well as the virtual interface.`

Reply:
That's fine, the redundancy was meant to reinforce the idea, but no biggie. Still would suggest replacing "as well as" with "and"

Reply:
"An IP address must be provided for each controller and the virtual interface."?

Reply:
@erinix Yep, agree.

Reply:
Sure!